internal/webserver/server.go

```diff
@@ -242,8 +242,11 @@ func (webserver *WebServer) secretHandler(writer http.ResponseWriter, req *http.
 	}
 
 	secretData.Path = strings.TrimSpace(secretData.Path)
-	if secretData.Path != "" && !strings.HasPrefix(secretData.Path, "/") {
+	if !strings.HasSuffix(webserver.Config.SecretStoreExclusive.Path, "/") && !strings.HasPrefix(secretData.Path, "/") {
```

Comment:
add some comments about why we are doing this. eg. remove extra `/` in the full URL path because Vault's secret APIs don't handle extra `/`.